docs/pages/modules/module-api.md

```diff
@@ -0,0 +1,391 @@
+---
+title: Module API
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+> Note: This API is still experimental and subject to change. Some features that you need may not be implemented yet.
```

Comment:
we should add some information about what we know is currently missing somewhere in this doc, eg: callbacks